scripts/hid_configurator/modules/dfu.py

```diff
@@ -376,7 +397,9 @@ def fwreboot(dev):
 
 
 def dfu_sync(dev):
-    success, fetched_data = dev.config_get('dfu', 'sync')
+    dfu_module_name = dev.get_module_variant_name_by_root('dfu')
```

Comment:
why returning only the variant suffix? we could return the whole name (that is "dfu/variant_name")
it seems to me that that would be easier to use

Reply:
We also use only variant_name for some purposes (compering with variant name during DFU). I thought it will be more readable that way.

Reply:
I think returning complete module name as @pdunaj suggested would be better approach here. Not only because it's simpler, but also it avoids constructing complete module name from name and variant in `dfu.py`. The `dfu.py` should not have information about naming format in `NrfHidDevice` class I think. It's not necessary here.

Reply:
I guess I am overvoted. So I will change it.  For me it is counter intuitive - Would not understand what is complete module name while reading this function. I will add comment explaining what is " complete module name".
Only variant name is as simple as with full name. After change instead of  constructing complete module name from name and variant in dfu.py you will have to deconstruct variant name in `configurator_cli.py`